mako/lookup.py

```diff
@@ -10,6 +10,7 @@
 import stat
 
 from mako import exceptions
+from mako import compat
```

Comment:
we actually don't need this import as "reraise" should be available as util.reraise().

Reply:
It's not available as `util.reraise`:

```
import mako.util
mako.util.reraise
Traceback (most recent call last):
  Python Shell, prompt 3, line 1
builtins.AttributeError: module 'mako.util' has no attribute 'reraise'
```